evergreen.yml

```diff
@@ -71,11 +71,6 @@ functions:
       params:
         command: ./venv/bin/pip3 install -vvv -i https://test.pypi.org/simple/ curatorbin==${version}
         directory: curatorbin
-    - command: shell.exec
```

Comment:
It seems like the `pypi` function is being used in a couple places. Is it okay to remove this?

Reply:
This is what I was trying to reference in the description; these never would have worked in the first place since looking at the git blame they have been shell.execs without a script since the beginning. I can try just changing these to script but if it's not a drop in replacement and tests fail then I think this needs to be a separate ticket.